starboard/common/test_main.cc

```diff
@@ -27,12 +26,19 @@ int InitAndRunAllTests(int argc, char** argv) {
 }
 }  // namespace
 
-// When we are building Evergreen we need to export SbEventHandle so that the
-// ELF loader can find and invoke it.
+#if defined(OS_STARBOARD)
```

Comment:
Let's use `BUILDFLAG(IS_STARBOARD)` unless there's an explicit reason for this file to not use a //build header

Reply:
The BUILDFLAG doesn't work for starboard/common as it doesn't depend on //base.

Building nplb gives the following error:
../../starboard/common/test_main.cc:29:5: error: function-like macro 'BUILDFLAG' is not defined
#if BUILDFLAG(IS_STARBOARD) 

The options I was considering were defined(STARBOARD) vs defined(OS_STARBOARD).
The third_party, that doesn't depend on //base has some variations of the OS_FOO. I see OS_ANDROID, ABSL_OS_ANDROID, ARMV8_OS_ANDROID.

I'm thinking defined(OS_STARBOARD) is decent choice here.

Reply:
BUILDFLAG is defined in //build (`#include "//build/build_config.h"`), not //base, so it should be fine to include in starboard

Reply:
Marking as unresolved

Reply:
Done